package com.day10;

public class ExceptionTest01 {

	public static void main(String[] args) {
		String[] message = {"one","two","three"};
//		for(String m : message) {
//			System.out.println(m);
//		}
		
//		for(int i=0; i<message.length; i++) {
//			System.out.println(message[i]);
//		}
		for(int i=0; i<10; i++) {
			System.out.println(message[i]);
			}
		String str = null;
		
		try {
		System.out.println(str.length());	
		}catch(NullPointerException n){
			System.out.println("Null 값"); // 예외 처리(Exception) try~catch로 사용
		}
		
		
	}

}
